<launch>
  <arg name="panther_version" />

  <node pkg="panther_manager" type="system_status_node.py" name="system_status_node"
    required="true" output="screen" />

  <node pkg="panther_manager" type="manager_bt_node" name="manager_bt_node" required="true"
    output="screen" if="$(eval panther_version >= 1.2)">
  </node>

</launch>